App/Controllers/NewsController.php

```diff
@@ -4,44 +4,71 @@
 
 use App\Controllers\Controller;
 use App\Models\News;
-use App\Requests\Request;
+use App\HttpRequest;
 use App\Http\JsonResponse;
 
 class NewsController extends Controller
 {
+    /**
+     * index
+     *
+     * @return void
+     */
     public function index()
```

Comment:
```suggestion
    public function index(): void
```

Return type can be defined in the code itself, docblock doesn't add anything else so just remove it.

(Same goes for your other doc blocks, don't feel like copy pasting this 20 times :^)